integration_test/README.md

```diff
@@ -261,7 +263,6 @@ The "Details" link takes you directly to the "logs" subdirectory to save you
         ├── syslog.txt
         └── systemctl_status_for_ops_agent.txt
 ```
-
```

Comment:
[Optional] I'd keep this blank line…

Reply:
Doesn't change the Markdown result, but I did it anyway in [46db9df](https://github.com/GoogleCloudPlatform/ops-agent/pull/1753/commits/46db9dfee78ae8468c868e5f14699160b2b79c78)